rfhub2/model/__init__.py

```diff
@@ -10,8 +10,7 @@ class Config:
         orm_mode = True
 
 
-class NestedKeyword(BaseModel):
-    id: int
+class KeywordUpdate(BaseModel):
```

Comment:
Why update method does not need id?

Reply:
This model is basically set of attributes that should be updateable on keyword record. Id is already passed in endpoint url so by removing it from this model we can easier update the record without any special consideration for attributes that should not be updateable.

Reply:
Ok, seems clever. I'm still trying to understand FastAPI framework.